metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1197,6 +1257,25 @@ private void resetState() {
      */
     private long newBytesSinceLastSnapshot = 0;
 
+    /**
+     * How long to delay partition leader balancing operations.
+     */
+    private final long leaderImbalanceCheckIntervalNs;
+
+    private static enum ImbalanceSchedule {
+        // Leader balancing has been scheduled
+        SCHEDULED,
+        // Leader balancing should be scheduled in the future
+        DEFERRED,
```

Comment:
If I understand correctly, DEFERRED is like our "idle" state here. That is, if no automatic preferred leader election is happening, we'll be in the DEFERRED state until an imbalance occurs. Is that right?

Reply:
Yes. I fixed the documentation for these values to make this clear.